src/utils.rs

```diff
@@ -1,12 +1,46 @@
 extern crate base64;
 
 use self::base64::encode;
+use http::retrieve_asset;
 use regex::Regex;
+use std::collections::HashMap;
 use url::{ParseError, Url};
 
+/// This monster of a regex is used to match any kind of URL found in a CSS
+/// stylesheet.
```

Comment:
I think "CSS stylesheet" sounds a bit redundant as CSS itself stands for "Cascading Style Sheets", maybe it's better to define it as "... URL found in CSS."